src/actions/pusher.js

```diff
@@ -0,0 +1,32 @@
+import * as pusher from '../api/pusher';
+import { getChannel } from '../utils/pusher';
+import { NOW_PLAYING } from '../constants/pods';
+import { getCurrentGameId } from '../selectors/game';
+import { nowPlayingLoaded } from '../actions/spotify';
+
+const PREFIX = 'PUSHER';
+
+export const CONNECTING_CLIENT = `${PREFIX}/CONNECTING_CLIENT`;
+export const UPDATING_CLIENTS = `${PREFIX}/UPDATING_CLIENTS`;
+export const SET_SYNCING = `${PREFIX}/SET_SYNCING`;
+
+export const setSyncing = isSyncing => ({ type: SET_SYNCING, isSyncing });
+
+export const connectToUpdates = channelId => async dispatch => {
+  console.log('%cConnecting to Pusher channel...', 'color: cyan');
```

Comment:
Is the body of this copied in from Peapod? Might be worth stubbing it out until we actually use it.

Reply:
It is. The original purpose of this branch was to add multiplayer using this implementation, but then I realized there were many things that should come before that in order of functionality. I can remove this stuff for now.